crates/oxc_linter/src/rules/eslint/getter_return.rs

```diff
@@ -119,72 +119,66 @@ impl GetterReturn {
     }
 
     /// Checks whether it is necessary to check the node
-    fn is_wanted_node(node: &AstNode, ctx: &LintContext<'_>) -> bool {
-        if let Some(parent) = ctx.nodes().parent_node(node.id()) {
-            match parent.kind() {
-                AstKind::MethodDefinition(mdef) => {
-                    if matches!(mdef.kind, MethodDefinitionKind::Get) {
-                        return true;
-                    }
+    fn is_wanted_node(node: &AstNode, ctx: &LintContext<'_>) -> Option<bool> {
```

Comment:
Since all `None` branch will return `false` finally, so it is safe to directly use `Option` with `?`